packages/expo/src/winter/runtime.native.ts

```diff
@@ -0,0 +1,26 @@
+// This file configures the runtime environment to increase compatibility with WinterCG.
+// https://wintercg.org/
+import { polyfillGlobal } from 'react-native/Libraries/Utilities/PolyfillFunctions';
```

Comment:
```suggestion
import { polyfillGlobal as defineGlobalProperty } from 'react-native/Libraries/Utilities/PolyfillFunctions';
```

since we are not polyfilling from an app here, we're defining the built-ins from the runtime